### ISSUE TITLE
"kernel" crashes

### ISSUE THREAD

**Issue by pzinn** [Macaulay2/M2#735]
Another day, another bug! I'm surprised this one went unnoticed for so long; could it have been introduced recently?

```
i1 : R=QQ[]

o1 = R

o1 : PolynomialRing

i2 : f=map(R,R)

o2 = map(R,R,{})

o2 : RingMap R <--- R

i3 : kernel f
-- SIGSEGV
-- stack trace, pid 21301:
level 0 -- return addr: 0x0x45c48f -- frame: 0x0x7ffff0b5bb60
level 1 -- return addr: 0x0x33b0560 -- frame: 0x0x5dbabc0
-- end stack trace
```

**Comment by DanGrayson** [comment]
No, that also crashes version 1.8.2.

Thanks!

**Comment by mikestillman** [comment]
Fixed in commit 8e9fa4415bf44.  Thanks for spotting this one!
